05/app.js

```diff
@@ -1,14 +1,47 @@
 const stats = {
     paragraphs: {
         'p1': 0,
+        'p2': 0,
+        'p3': 0,
     },
     links: {
         '/dolor.html': 0,
     }
 };
 
 /* tutaj umieść swój kod */
+// runkcja do zliczania kliknięć w linki
+function countLinkClicks(event) {
+    event.preventDefault();
+    const link = event.currentTarget.getAttribute('href');
 
+    if (link && stats.links.hasOwnProperty(link)) {
+        stats.links[link]++;
+        fireCustomEvent(statsElement, 'render');
+    }
```

Comment:
Powinno być odwrotnie. Jeśli element nie istnieje to trzeba go utworzyć tj. `stats.link[link] = 0` i potem zwiększyć o jeden. Jeśli istnieje to tylko zwiększyć o 1 :)

PS. Nie trzeba uruchamiać `fireCustomEvent` bo jest tor już robione w linii 84 :)